status-im/status-react
# [ISSUE #1835] Migrate to unique toolbar implementation

addresses #1835

### Summary:

Migrate all toolbars to `toolbar/toolbar`

### Steps to test:

No fix nor new features: app needs to be sanity-checked.

- Open Status
- Check all toolbars look ok

status: ready
